src/drive/targets/manifest.webapp

```diff
@@ -175,5 +175,12 @@
       "verbs": ["POST"],
       "description": "Remote-doctype required to send anonymized measures to the DACC shared among mycozy.eu's Cozy."
     }
+  },
+  "accept_from_flagship": true,
+  "accept_documents_from_flagship": {
+    "accepted_mime_types": ["*/*"],
+    "max_number_of_files": 1,
+    "max_size_per_file_in_MB": 10,
+    "route_to_upload": "/#/create?fromFlagshipUpload=true"
```

Comment:
@Ldoppea did we update the documentation of the manifest.webapp ? https://docs.cozy.io/en/cozy-stack/apps/#the-manifest

Reply:
Done here : https://github.com/cozy/cozy-stack/pull/4179